image-types/rhel8/amazon-ec2.md

```diff
@@ -0,0 +1,49 @@
+# Amazon EC2 Image
+
+Owner: *None*
+
+This image is meant to be used in [Amazon Elastic Compute Cloud (EC2)][ec2], a
+popular cloud computing platform. It has to follow Amazon’s
+[guidelines][guidelines] and [requirements][requirements] for shared AMIs.
```

Comment:
Is the intention that we duplicate the parts of the guidelines/requirements that we choose to adopt in this doc, so that if they develop over time, we would make the explicit choice to copy the changes here?

Reply:
I think so, yes. I don't think it makes sense to copy everything over straight away. There's a lot of leeway in those documents, though. We should document the choices we're making and the reasons for them here.